src/dropbox.js

```diff
@@ -811,7 +837,7 @@
           return Promise.reject(new Error("Cannot delete '" + path + "': too many files involved"));
         }
 
-        if (resp.status === 200) {
+        if (resp.status === 200 || resp.status === 404) {
```

Comment:
Why is this now checking for 404 as well as 200? I think there should be a comment about this because it is not clear to me